src/index.css

```
@@ -8,6 +9,331 @@
+  margin: 10px;
+}
+
+.like-button {
+  min-width: 44px;
+  min-height: 44px;
+  border: 2px solid; 
+  border-color: #385170;
+  border-radius: 50%;
+  background: #ececec;
+  color: #385170;
+  border: none;
```

Biome

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)